### DISCUSSION TITLE
OSC 9;9 support for Zsh

### DISCUSSION THREAD

**Discussion by patrick-5546** [JanDeDobbeleer/oh-my-posh#1508] [Troubleshoot] [answered]
I am trying to use the same theme for both my PowerShell and Zsh profiles in Windows Terminal. Adding `"osc99": true` to my theme allows my PowerShell profile to open new tabs/panes in the current working directory. However, this causes my Zsh profile to open new tabs/panes in the root directory. When I omit the line, my Zsh profile works, but PowerShell doesn't. Any ideas for what I can do, besides creating a theme for each profile?

Note: to get my Zsh profile to work, I also had to add the code from https://github.com/microsoft/terminal/issues/3158#issuecomment-986826132 to my `~/.zshrc`

**Comment by JanDeDobbeleer** [comment]
~~That's a terminal/shell feature, not something oh-my-posh can fix or add support for.~~

Wait, the original comment wasn't fully loaded. I'll check the thread and see if there's something we can do.

**Comment by lnu** [comment, marked as answer]
https://github.com/JanDeDobbeleer/oh-my-posh/pull/1516

**Reply by patrick-5546** [reply]
Thanks, v6.42.2 fixed it for me!
